core/io/marshalls.cpp

```diff
@@ -30,7 +30,10 @@
 
 #include "marshalls.h"
 
+#include "core/core_string_names.h"
+#include "core/io/resource_loader.h"
 #include "core/object/ref_counted.h"
+#include "core/object/script_language.h"
```

Comment:
Are you using ScriptLanguage here, or just Script?

Reply:
Just `Script`.

Reply:
Ah I thought Script was in its own file, but it's ScriptInstance that we recently split.